heron/instance/src/python/utils/metrics/py_metrics.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, metrics_collector):
                     PY_GC_GENERATION_1_THRESHOLD: self.g1_threshold,
                     PY_GC_GENERATION_2_THRESHOLD: self.g2_threshold,
                     PY_GC_GENERATION_3_THRESHOLD: self.g3_threshold}
-    super(PyMetrics, self).__init__(self.metrics)
+    _ = super()
```

Comment:
Should this have been `super().__init__(self.metrics)`? I don't know Python well enough, but that was the recommendation I saw online. I'm not familiar with the `_ = super()` syntax.

Reply:
This syntax is analogous to what we have in Golang. I am essentially telling the Python3 interpreter and linter know "Yes, I am aware a value is returned from this function call. No, I do not need it, and yes I wish to ignore it."

Some linters will output a warning about this and it could potentially become an error down the road. Let us not generate/leave technical debt to be dealt with later.

Reply:
I stand corrected, if calling `super().__init__` there is no return value. My IDE's linter was reporting warnings in other calls to `super()`.